test/parallel/test-npm-install.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const common = require('../common');
+
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+const fs = require('fs');
+
+common.refreshTmpDir();
+
+const npmPath = path.join(
+  common.testDir,
+  '..',
+  'deps',
+  'npm',
+  'bin',
+  'npm-cli.js'
+);
+
+const args = [
+  npmPath,
+  'install'
+];
+
+const pkgContent = '{}';
+
+const pkgPath = path.join(common.tmpDir, 'package.json');
+
+fs.writeFile(pkgPath, pkgContent, testNpm);
+
+function testNpm(err) {
+  assert.ifError(err);
+
+  const proc = spawn(process.execPath, args, {
+    cwd: common.tmpDir
+  });
+
+  proc.on('exit', function(code) {
```

Comment:
Wrap in `common.mustCall()` and also check that `signalCode` (the second argument) is null.

(Also, I'd use strict equality checks.)